### PR TITLE
#170764595 Add accommodation

### PR DESCRIPTION
**Pull request by shaazk**
### What does this PR do?
- Enables a Host-supplier to add accommodation facilities
- Enables to a requester to view approved accommodations
- Enables a user to view all or a specific accommodation
### Description of Task to be completed?
- add accommodation facilities table
- add functionality to ensure Host-supplier can add accommodation facilities
- add functionality to enable image upload to Cloudinary
### How should this be manually tested?
- git clone this repo and check out to ft-accomodation-facility-168050022
- run npm i to install all the dependencies
- run sequelize db:migrate:undo:all then npm run migrate to add the accommodations schema
- run the application with npm run dev
- register a user in the system
- update their role_value in the database manually in the seeders. ensure their account is verified as well.
- login the user and user the token provided in subsequent operations
- send a POST request to /accommodations in postman via a form-data as shown in the image below:]
Screenshot from 2020-02-13 02-20-00

### What are the relevant pivotal tracker stories?
[#170764595](https://www.pivotaltracker.com/story/show/170764595)

### Screenshots (if appropriate)
Screenshot from 2020-02-13 02-24-52
Screenshot from 2020-02-13 02-24-38

Screenshot from 2020-02-12 22-34-46

Screenshot from 2020-02-12 22-37-32

### CODE REVIEW

**Review by Alexander96779 — APPROVED** [review]
Way to Go Sharon

File: src/controllers/accommodation.controller.js
Comment (istafrica-dennw) [inline src/controllers/accommodation.controller.js]: Put your class name in singular

File: src/controllers/accommodation.controller.js
Comment (istafrica-dennw) [inline src/controllers/accommodation.controller.js:39]: Do we need an approval @shaazk ? if yes from who?

File: src/routes/accommodation.route.js
Comment (istafrica-dennw) [inline src/routes/accommodation.route.js]: Hi @shaazk Even the travel admin can add an accommodation. read well your PT story. Thank you.

**Review by istafrica-dennw — CHANGES_REQUESTED** [review]

File: src/controllers/accommodation.controller.js
Comment (shaazk) [reply src/controllers/accommodation.controller.js:39]: Yes, the super admin has to approve the accommodation request from the host-supplier like the description suggests, there will have to be a background check of an accommodation before it is approved so that is what i implemented

File: src/routes/accommodation.route.js
Comment (shaazk) [reply src/routes/accommodation.route.js]: Let me fix it.

**Review by ajimae — APPROVED** [review]
LGTM!

**Pull request merged**
